src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+$day-size: 100px;
+$gap: 1px;
+
+.calendar {
+  max-width: calc($day-size * 7 + $gap * 6);
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+  &__day {
+    background-color: #eee;
+    position: relative;
+    box-sizing: border-box;
+    width: $day-size;
+    height: $day-size;
+    border: 1px solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &:hover {
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+      cursor: pointer;
+      transition-duration: 0.5s;
+    }
+  }
+}
+
+@for $i from 1 to 32 {
+  .calendar :nth-child(#{$i}):after {
+    content: '#{$i}';
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+  }
```

Comment:
The selector `.calendar :nth-child(#{$i}):after` may not work as intended because the space before `:nth-child` makes it a descendant combinator. This means it will select any element that is the nth child of its parent, not necessarily the direct children of `.calendar`. Consider removing the space to target direct children.